tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -16,9 +16,8 @@ class ViewImage(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == 'Gallery')
+        self.wait_to_be_displayed()
```

Comment:
Ok, how does that work then? I don't understand.
I see mentions of view_image here: http://mxr.mozilla.org/gaia/search?string=view_image
Nothing seems related to Gallery.
But I guess because this file is in a subdirectory of galllery, it somehow is the constructor of Gallery?